RELEASE-NOTES.txt

```diff
@@ -1,6 +1,10 @@
 *** PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
 *** Use [*****] to indicate smoke tests of all critical flows should be run on the final IPA before release (e.g. major library or OS update).
 
+20.2
```

Comment:
Note: Since the tooling for code freeze adds a section for the next release automatically, it's necessary to check before merging that we don't have duplicated sections before merging to trunk.